workout.py

```diff
@@ -1,21 +1,100 @@
 '''Workout Planning App by Zach and Jake. Name TBD, TXST SWE, EIR Laith Hasanian.'''
-import random
 import os
-from urllib.parse import urlparse, urljoin
+from datetime import date
 import requests
 from flask import Flask, render_template, request, redirect, url_for, session, flash
 from flask_sqlalchemy import SQLAlchemy
-from flask_login import LoginManager, UserMixin, login_user, login_required, logout_user, current_user
+from flask_login import LoginManager, UserMixin, login_user, login_required, logout_user
 from dotenv import load_dotenv, find_dotenv
 
 load_dotenv(find_dotenv())
+workout = {}
+workout['today'] = ''
 
 app = Flask(__name__)
+app.secret_key = 'super_secret_secret_key'
+app.config['SQLALCHEMY_DATABASE_URI'] = os.getenv('DB_URL')
+db = SQLAlchemy(app)
+
+class People(UserMixin, db.Model):
+    '''Table of Users'''
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(40), unique=True)
+
+    def __repr__(self) -> str:
+        return self.username
+
+class Workouts(db.Model):
+    '''Table listing the target muscle(s) for a user in a workout session, and date.'''
+    username = db.Column(db.String(40), primary_key=True)
+    targets = db.Column(db.String(200))
```

Comment:
Just in case users add a lot of workouts, the length of targets in the db might need to be longer, not a real issue though but maybe something to change later on.